Source/NomadApp.cpp

```
@@ -0,0 +1,1710 @@
+#include "AppLifecycle.h"
+#include "ServiceLocator.h"
+#include "AudioThreadConstraints.h"
+#include "../NomadUI/Graphics/OpenGL/NUIRendererGL.h"
+#include "../NomadUI/Core/NUIDragDrop.h"
+#include "../NomadAudio/include/AudioGraphBuilder.h"
+#include "../NomadAudio/include/AudioCommandQueue.h"
+#include "../NomadAudio/include/AudioRT.h"
+#include "../NomadAudio/include/PreviewEngine.h"
+#include "../NomadAudio/include/PluginManager.h"
+#include "../NomadCore/include/NomadLog.h"
+#include "../NomadAudio/include/PluginManager.h"
```

Clang (14.0.6)

[warning] duplicate include

(readability-duplicate-include)